lms/djangoapps/instructor/views/api.py

```diff
@@ -1054,7 +1054,8 @@ def get_problem_responses(request, course_id):
     try:
         for problem_location in problem_locations.split(','):
             problem_key = UsageKey.from_string(problem_location).map_into_course(course_key)
-    except InvalidKeyError:
+    except InvalidKeyError as err:
+        log.error("ProblemResponsesError: %s", text_type(err))
```

Comment:
log.exception would be better as it will log stack too(but if not useful for the current scenario, it is ok)